turbo.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://turbo.build/schema.json",
   "ui": "tui",
   "globalEnv": ["__FIREFOX__"],
+  "daemon": false,
```

Comment:
Are there any performance issues? I'm curious about the tradeoffs of disabling the daemon.

Reply:
I didn't have any, but the main reason i've did it is, issue with no possibility of removal `.turbo` on Windows, with this i don't need to stop it from `package.json` script